gulpfile.js

```diff
@@ -139,7 +77,7 @@ function browsersync() {
     // @link http://www.browsersync.io/docs/options/
 
     // Project URL.
-    proxy: projectURL,
+    proxy: config.projectURL,
```

Comment:
You defined it as `CONFIG` but are using it as `config` that won't work.

Reply:
Fixed. My fault. I got my branches crossed up - literally.